forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -7476,6 +7501,15 @@ public List<SpellAbility> getAllPossibleAbilities(final Player player, final boo
             }
         }
 
+        if (isInPlay() && isRoom() && !isPhasedOut() && player.canCastSorcery()) {
```

Comment:
I'm starting to think checking `isRoom` isn't correct:
> 116.2m. A player who controls a permanent that has one or more locked halves (see rule 709.5) may pay the mana cost of a locked half of that permanent to give that permanent the appropriate unlocked designation.

So it should still work when enchanted by _Minimus Containment_?
Or only if it doesn't use `RemoveAllAbilities$`? 🤔

Reply:
That's probably for rules manager to asked

Reply:
Also Special Action might not be removed by that

Reply:
fixed, it doesn't need to check for Room anymore